tufaceous-lib/src/key.rs

```diff
@@ -78,7 +59,9 @@ impl FromStr for Key {
 
     fn from_str(s: &str) -> Result<Key> {
         match s.split_once(':') {
-            Some(("ed25519", hex)) => Ok(Key::Ed25519(FromHex::from_hex(hex)?)),
+            Some(("ed25519", base64)) => {
+                Ok(Key::Ed25519 { pkcs8: URL_SAFE.decode(base64)? })
```

Comment:
Swapping from hex to base64 -- is this a problem for existing serialization? Do we have any stringified keys stored anywhere?

Reply:
It also changes from being the bare private key to a PKCS8 document containing both the private and public keys, so it's a format change either way. aws-lc-rs provides no mechanism to load only a private key anymore. (AIUI this is because there are different ways you could convert a private key into a public key?)

I am fairly certain we don't have any keys of importance stored anywhere unless folks are testing with a specific set of keys. The keys used to generate the repo in CI are generated on the fly and thrown away.

Reply:
Yeah, I think it's only used for ephemeral keys and local testing.